api/v1/tetragon/events.proto

```diff
@@ -42,6 +42,9 @@ message Filter {
     // Filter by process.arguments field using RE2 regular expression syntax:
     // https://github.com/google/re2/wiki/Syntax
     repeated string arguments_regex = 7;
+    // Filter events by pod labels using Kubernetes label selector syntax:
+    // https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#label-selectors
```

Comment:
```
+ If  a labels is defined, events without Pod information will _always_ be filtered out.
```

(see other comment)

Reply:
yeah let me add a separate documentation for the current filter syntax

Reply:
updated the comment. i'll follow up with a separate PR to document filter syntax once this PR gets merged.